core/src/zeit/push/workflow.py

```diff
@@ -72,6 +73,26 @@ def delete(self, query):
         self.message_config = tuple(config)
 
 
+@grok.subscribe(
+    zeit.cms.content.interfaces.ICommonMetadata, zeit.cms.checkout.interfaces.IBeforeCheckinEvent
+)
+def update_content_channel_from_template(context, event):
+    push = zeit.push.interfaces.IPushMessages(context)
+    source = zeit.push.interfaces.PAYLOAD_TEMPLATE_SOURCE
+    channels = set()
+    for config in push.message_config:
+        name = config.get('payload_template')
+        if name:
+            template = source.find(name)
+            for channel in template.channels:
+                if channel not in context.channels:
+                    channels.add(channel)
+
+    if not channels:
+        return
+    context.channels += tuple(channels)
```

Comment:
```suggestion
    if channels:
	context.channels += tuple(channels)
```
Ich bin ja sehr für Guard-Pattern, aber in so einem Fall würd ichs tatsächlich nicht verwenden...